web-api/package.json

```diff
@@ -45,6 +46,7 @@
     "@types/async-retry": "1.4.2",
     "@types/humps": "2.0.0",
     "@types/jest": "26.0.13",
+    "@types/jsonwebtoken": "^8.5.0",
```

Comment:
Should this be a pinned version?

Reply:
Yes, I fixed it!
https://github.com/jubilee-works/timetree-sdk-js/pull/250/commits/d6a1f49ef9bcd02a45c6c1fa82680b3dfa308e6f